cmd/cstor-pool-mgmt/pool/pool.go

```diff
@@ -183,6 +184,7 @@ func CheckForZreplContinuous(ZreplRetryInterval time.Duration) {
 			time.Sleep(ZreplRetryInterval)
 			continue
 		}
+		glog.Errorf("Error in CheckForZreplContinuous : %v", err)
```

Comment:
Please re-phrase the log message to something like "zpool status returned error in CheckForZreplContinuous: %v"